staking/contracts/contracts.py

```diff
@@ -315,5 +345,5 @@ def config(
         f.write(clearstate)
 
     with open("pyteal_abi.json", "w") as f:
-        f.write(json.dumps(abi.dictify()))
+        f.write(json.dumps(contract.dictify()))
```

Comment:
```suggestion
        f.write(json.dumps(contract.dictify(), indent=4))
```
nit: a more human readable contract